test/polyrith.lean

```diff
@@ -37,8 +38,8 @@ meta def tactic.test_polyrith (only_on : bool) (hyps : list pexpr)
   guard (out = expected_out) <|>
     fail!"expected final output: {expected_out}\nbut produced: {out}"
 
-meta def format_string_list (input : list string) : string :=
-to_string $ input.map (λ s, "\"" ++ s ++ "\"")
+meta def format_string_list (input : list string) : format :=
+"[" ++ (format.join $ (input.map (λ s, ("\"" : format) ++ format.of_string s ++ "\"")).intersperse ("," ++ format.line)) ++ "]"
```

Comment:
Arguably nicer as:
```suggestion
meta def format_string_list (input : list string) : format :=
"[" ++ format.nest 1 (format.join $ (input.map (λ s, ("\"" : format) ++ format.of_string s ++ "\"")).intersperse ("," ++ format.line)) ++ "]"
```
So that the output is
```
  ["ff",
   "int",
```
not
```
  ["ff",
  "int",
```

Reply:
When the VSCode extension fix lands, these changes will be easier and I might feel more willing! Right now it's not worth it.